apollo-ios/Sources/Apollo/SelectionSet+JSONInitializer.swift

```diff
@@ -16,6 +16,7 @@ extension RootSelectionSet {
   ///                the given JSON response data.
   public init(
     data: JSONObject,
+    fulfilledFragments: FulfilledFragments = .labels([]),
```

Comment:
We may want to leave this parameter out of here depending on whether the defer preview release ships with support for selection set initializers or not. If it doesn't then it might be confusing to users of what this actually does.